.github/workflows/ci.yml

```diff
@@ -7,6 +7,27 @@ on:
   workflow_dispatch:
 
 jobs:
+  lint:
+    name: "nph Lint"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2  # In PR, has extra merge commit: ^1 = PR, ^2 = base
+
+      - name: Check nph formatting
+        # Pin nph to a specific version to avoid sudden style differences.
+        # Updating nph version should be accompanied with running the new
+        # version on the project directory.
+        run: |
+          VERSION="v0.5.1"
+          ARCHIVE="nph-linux_x64.tar.gz"
+          curl -L "https://github.com/arnetheduck/nph/releases/download/${VERSION}/${ARCHIVE}" -o ${ARCHIVE}
+          tar -xzf ${ARCHIVE}
```

Comment:
Perhaps it should instead become a nimble dependency? Did you tried if that works?

Reply:
Sure, yes it works locally for me. I'll try that